DistributionPackages/Neos.MarketPlace/Resources/Public/Assets/feed.xsl

```diff
@@ -0,0 +1,172 @@
+                            ". footer .";
+                        padding: 1rem 0;
+                        font-size: 16px;
+                    }
+
+                    .site-header {
+                        display: grid;
+                        grid-gap: 1rem;
+                        grid-area: header;
+                        grid-template-areas:
+                            "logo"
+                            "title"
+                            "description";
+                        padding-bottom: 1rem;
+                        border-bottom: 1px solid #eee;
+                    }
+
+                    @media (min-width: 768px) {
+                        .site-header {
+                            grid-template-areas:
+                                "title logo"
+                                "description logo";
+                        }
+                    }
+
+                    a {
+                        color: #00adee;
+                        text-decoration: none;
+                    }
+
+                    a:hover {
+                        text-decoration: underline;
+                    }
+
+                    h1 {
+                        color: #3b444f;
+                        grid-area: title;
+                        margin: 0;
+                    }
+
+                    .logo {
+                        margin: 1em;
+                        grid-area: logo;
+                        max-width: 250px;
+                        height: auto;
+                        align-self: center;
+                        justify-self: center;
+                    }
+
+                    @media (min-width: 768px) {
+                        .logo {
+                            justify-self: flex-end;
+                        }
+                    }
+
+                    .description {
+                        grid-area: description;
+                        margin: 0;
+                    }
+
+                    main {
+                        grid-area: feed;
+                    }
+
+                    article + article {
+                        margin-top: 1rem;
+                        border-top: 1px solid #eee;
+                        padding-top: 1rem;
+                    }
+
+                    article h2 {
+                        color: #00adee;
+                        margin: 0 0 .5rem;
+                    }
+
+                    article p {
+                        margin-bottom: 1em;
+                    }
+
+                    .entry__details {
+                        font-size: 90%;
+                        color: #656a71;
+                    }
+
+                    .site-footer {
+                        grid-area: footer;
+                        padding-top: 1rem;
+                        border-top: 1px solid #eee;
+                    }
+                </style>
+            </head>
+            <body style="font-family:'Work Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;">
```

Comment:
What do you think about
```suggestion
            <body style="font-family:-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"">
```
Some more infos: https://css-tricks.com/snippets/css/system-font-stack/

Reply:
Actually this css should also go into the top declaration. 

What are those fonts for?

Reply:
Ah sorry, now saw your link